api/src/Microsoft.Azure.IIoT.OpcUa.Api.Publisher/src/Models/DiagnosticInfoApiModel.cs

```diff
@@ -4,128 +4,128 @@
 // ------------------------------------------------------------
 
 namespace Microsoft.Azure.IIoT.OpcUa.Api.Publisher.Models {
-    using System.Runtime.Serialization;
     using System;
+    using System.Runtime.Serialization;
 
     /// <summary>
     /// Model for a diagnostic info response.
     /// </summary>
     [DataContract]
     public class DiagnosticInfoApiModel {
- 
+
         /// <summary> EndpointInfo /// </summary>
-        [DataMember(Name = "EndpointInfo", Order = 0,
-            EmitDefaultValue = false)]
-        public PublishNodesEndpointApiModel EndpointInfo { get; set; }
+        [DataMember(Name = "endpoint", Order = 0,
+            EmitDefaultValue = true)]
+        public PublishNodesEndpointApiModel Endpoint { get; set; }
```

Comment:
Let's update the documentation accordingly.

Reply:
absolutely. will do in a separate PR after https://github.com/Azure/Industrial-IoT/pull/1582 is merged

Reply:
PR is merged now.